crates/sui-config/src/node.rs

```diff
@@ -241,12 +241,14 @@ pub fn default_zklogin_oauth_providers() -> BTreeMap<Chain, BTreeSet<String>> {
         "Credenza3".to_string(),
         "AwsTenant-region:us-east-1-tenant_id:us-east-1_LPSLCkC3A".to_string(),
         "AwsTenant-region:us-east-1-tenant_id:us-east-1_qPsZxYqd8".to_string(),
+        "Ambrus".to_string(),
```

Comment:
since ambrus is already enabled in devnet, it should not be added here, it's already present as an aws tenant on line 253. 

instead, you just need to add what's in line 243 (what ambrus tenant id encodes to) to the providers list below on line 251 (what the mainnet and testnet config is)

Reply:
Thank you @joyqvq . 
I have changed it.